DOC: Add metric prcp partitioning segmentation and sign in math/num (#122)
DassHydro/smash: doc/source/math_num_documentation/efficiency_error_metric.rst

.. _math_num_documentation.efficiency_metric:

=======================
Efficiency/Error Metric
=======================

> The aim of this section is to present all the efficiency/error metrics that can be used as part of the observation term in the inverse problem.

Reviewer: Je pense qu'on pourrait compléter comme ça:
```suggestion
The aim of this section is to present all the efficiency/error metrics that can be used to calibrate the model and evaluate its performance in simulating discharges.
```

> Denote :math:`Q` and :math:`Q^*` the simulated and observed discharge respectively and :math:`t\in[1..T]` a time step.

Reviewer: ```suggestion
Denote :math:`Q` and :math:`Q^*` the simulated and observed discharge, respectively, with :math:`t\in[1..T]` representing a time step for each.
```

> nse

Reviewer: On écrit plutôt en majuscule dans cette partie? 
```suggestion
NSE
```

---

The Nash-Sutcliffe Efficiency

.. math::

    j_{nse} = 1 - \frac{\sum_{t=1}^{T}\left(Q(t) - Q^*(t)\right)^2}{\sum_{t=1}^{T}\left(Q^*(t) - \mu_{Q^*}\right)^2}

with :math:`\mu_{Q^*}` the mean of the observed discharge.

> nnse

Reviewer: Pareil?
```suggestion
NNSE
```

----

The Normalized Nash-Sutcliffe Efficiency

.. math::

    j_{nnse} = \frac{1}{2 - j_{nse}}

> kge

Reviewer: ```suggestion
KGE
```

---

The Kling-Gupta Efficiency

.. math::

    j_{kge} = 1 - \sqrt{(r - 1)^2 + (\alpha - 1)^2 + (\beta - 1)^2}

> with :math:`r` the Pearson correlation coefficient, :math:`\alpha` the term representing the variability of prediction errors and 

Reviewer: ```suggestion
with :math:`r` the Pearson correlation coefficient, :math:`\alpha` the variability of prediction errors, and 
```

:math:`\beta` the bias term. They are defined as follows:

.. math::
    :nowrap:

    \begin{eqnarray}

        &r& &=& &\frac{\text{cov}(Q, Q^*)}{\sigma_Q \sigma_{Q^*}}\\
        &\alpha& &=& &\frac{\sigma_Q}{\sigma_{Q^*}}\\
        &\beta& &=& &\frac{\mu_Q}{\mu_{Q^*}}

    \end{eqnarray}

> with :math:`\text{cov}(Q, Q^*)` the covariance between :math:`Q` and :math:`Q^*`, :math:`\mu_{Q}` and :math:`\mu_{Q^*}` the mean of the simulated and observed discharge respectively and 

Reviewer: ```suggestion
with :math:`\text{cov}(Q, Q^*)` the covariance between :math:`Q` and :math:`Q^*`, :math:`\mu_{Q}` and :math:`\mu_{Q^*}` the mean of the simulated and observed discharge, respectively, and 
```

> :math:`\sigma_{Q}` and :math:`\sigma_{Q^*}` the standard deviation of the simulated and observed discharge respectively.

Reviewer: ```suggestion
:math:`\sigma_{Q}` and :math:`\sigma_{Q^*}` the standard deviation of the simulated and observed discharge, respectively.
```

> mae

Reviewer: ```suggestion
MAE
```

---

The Mean Absolute Error

.. math::

    j_{mae} = \frac{1}{T} \sum_{t=1}^T \lvert Q(t) - Q^*(t) \rvert

> mape

Reviewer: ```suggestion
MAPE
```

----

The Mean Absolute Percentage Error

> .. Can't \left \lvert \right \rvert ...
> 

Reviewer: Cette ligne a été écrit par erreur?
```suggestion
```

Author: Non j'ai mis en commentaire parce que je compile la doc en faisant \left \lvert ... \right \rvert, j'ai pas d'affichage ... Du coup, visuellement on a une valeur absolue qui prend pas toute la fraction. Mais peu importe, c'est pas tres grave

.. math::

    j_{mape} = \frac{1}{T} \sum_{t=1}^T \lvert \frac{Q(t) - Q^*(t)}{Q^*(t)} \rvert

> mse

Reviewer: ```suggestion
MSE
```

---

The Mean Squared Error

.. math::

    j_{mse} = \frac{1}{T} \sum_{t=1}^T \left(Q(t) - Q^*(t)\right)^2

> rmse

Reviewer: ```suggestion
RMSE
```

----

The Root Mean Squared Error

.. math::

    j_{rmse} = \sqrt{j_{mse}}

> lgrm

Reviewer: ```suggestion
LGRM
```
